.gitignore

```diff
@@ -3,4 +3,4 @@ npm-debug.log
 build
 generated
 .antwar
-
+.vscode
```

Comment:
Also, please let me know if any other adjustments are needed. I was a bit hesitant to add examples but thought they were appropriate where I did (hopefully).